electron/ui/src/views/Optimization/Descriptions.js

```diff
@@ -32,40 +32,47 @@ export const descriptions = {
               Select the solver you would like to use. Note: Gurobi requires a license. 
               If you do not have a Gurobi licence, select "CBC", an open source solver.
             </div>,
-    scaleModel: <div>
+    scale_model: <div>
             Choose whether you would like to scale the model or not.
           </div>,
 
-    pipelineCapacity: <div>
+    pipeline_capacity: <div>
         Alternate pipeline capacity selection:<br/>
         -Input: use input for pipeline capacity<br/>
         -Calculated: calculate pipeline capacity from pipeline diameters</div>,
-    nodeCapacity: <div>
+    pipeline_cost: <div>
+        Alternate pipeline capex cost structures (distance or capacity based):<br/>
+        -Capacity-based: use pipeline capacities and rate in [currency/volume] to calculate pipeline CAPEX costs<br/>
+        -Distance-based: use pipeline distances and rate in [currency/(diameter-distance)] to calculate pipeline CAPEX costs</div>,
+    node_capacity: <div>
```

Comment:
Is this redundant in light of lines 7-10 above?

Reply:
yeah I changed the key from `pipelineCost` to `pipeline_cost`, then I guess I overlooked that there had already been a description. just pushed a change to remove the redundant one above